pkg/types/types.go

```diff
@@ -14,11 +14,7 @@ type Index struct {
 	GroupID     string
 	ArtifactID  string
 	Version     string
+	Classifier  string
```

Comment:
You don't use this field to build DB.
https://github.com/aquasecurity/trivy-java-db/blob/793673c0d6d697fb86dac0fb1cf5ecc29bdd0e99/pkg/db/db.go#L126

Reply:
Added a comment
https://github.com/aquasecurity/trivy-java-db/pull/57/commits/72bbd7bcb6bd72633596f3bb0df3cdfc1bf4a193

Reply:
we must not forget that until we add this, the database will not include new artifacts with classifiers.